src/Charcoal/Admin/Widget/QuickFormWidget.php

```diff
@@ -10,6 +10,41 @@
  */
 class QuickFormWidget extends ObjectFormWidget
 {
+    /**
+     * Ident for tab display.
+     *
+     * @const string
+     */
+    const DISPLAY_MODE_TAB = 'tab';
+    /**
+     * Ident for lang tab display.
+     *
+     * @const string
+     */
+    const DISPLAY_MODE_LANG = 'lang';
+
+    /**
+     * Whether to display the language switcher.
+     *
+     * @var boolean
+     */
+    protected $showLanguageSwitch;
+
+    /**
+     * @param  array $data The widget data.
+     * @return FormWidget Chainable
```

Comment:
With future versions of PHP, this will be type-hinted with `self`.

```suggestion
     * @return self
```